src/main/java/stsjorbsmod/cards/cull/deckoftrials/DemonicCoup.java

```diff
@@ -35,6 +37,11 @@ public void use(AbstractPlayer p, AbstractMonster m) {
         addToBot(new ApplyPowerAction(p, p, new RitualPower(p, RITUAL_AMOUNT, true)));
```

Comment:
I think you should be setting this to `magicNumber` instead of `RITUAL_AMOUNT`. I think this will always apply 2 ritual, even if it's upgraded.

Reply:
I didn't even change that, but it should absolutely be magicNumber, I'll make that change